string GPS_location
---
string weather_response
